<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow1</class>
 <widget class="QMainWindow" name="MainWindow1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1862</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1831</width>
      <height>911</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>main page</string>
     </attribute>
     <widget class="QTableView" name="tab_mat">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>90</y>
        <width>481</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>570</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_modifier_2">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>570</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>570</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>ID_Materiel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>date</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Designation</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ID_MATERIELS">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="DATE_M">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>350</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>410</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix</string>
      </property>
     </widget>
     <widget class="QPushButton" name="displayTableButton">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>280</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>show table</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_pdf">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>120</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Imprimer">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>200</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>30</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>20</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_chart">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>50</y>
        <width>701</width>
        <height>391</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="statetat">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>370</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Stat etat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="statDESIGNATION">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>430</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Stat Designation</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_chart2">
      <property name="geometry">
       <rect>
        <x>605</x>
        <y>120</y>
        <width>671</width>
        <height>341</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>630</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AscButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>680</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>ASC</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DescButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>680</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>DESC</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Refrech</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_chart_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>761</width>
        <height>431</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lab_chart2_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>841</width>
        <height>461</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1911</width>
        <height>1031</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/img/img/1.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>EMPLACEMENT</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="EMPLACEMENT">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>270</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>490</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Fournisseur</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="FOURNISSEUR">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>500</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>570</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Etat</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PRIX_M">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>420</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="ETAT">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>580</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>En stock</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>En cours</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hors service</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="DESIGNATION">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>DESIGNATION1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DESIGNATION2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DESIGNATION3</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="sendNotification">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>730</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Reservé</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>690</y>
        <width>311</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>780</y>
        <width>118</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushsegment">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>630</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>7 seg</string>
      </property>
     </widget>
     <zorder>timeEdit</zorder>
     <zorder>label_3</zorder>
     <zorder>lab_chart2_2</zorder>
     <zorder>lab_chart_2</zorder>
     <zorder>lab_chart2</zorder>
     <zorder>lab_chart</zorder>
     <zorder>tab_mat</zorder>
     <zorder>pb_ajouter</zorder>
     <zorder>pb_modifier_2</zorder>
     <zorder>pb_supprimer</zorder>
     <zorder>label_2</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
     <zorder>ID_MATERIELS</zorder>
     <zorder>DATE_M</zorder>
     <zorder>label_11</zorder>
     <zorder>displayTableButton</zorder>
     <zorder>pb_pdf</zorder>
     <zorder>Imprimer</zorder>
     <zorder>lineEdit</zorder>
     <zorder>label_12</zorder>
     <zorder>statetat</zorder>
     <zorder>statDESIGNATION</zorder>
     <zorder>label</zorder>
     <zorder>AscButton</zorder>
     <zorder>DescButton</zorder>
     <zorder>pushButton</zorder>
     <zorder>label_7</zorder>
     <zorder>EMPLACEMENT</zorder>
     <zorder>label_8</zorder>
     <zorder>FOURNISSEUR</zorder>
     <zorder>label_9</zorder>
     <zorder>PRIX_M</zorder>
     <zorder>ETAT</zorder>
     <zorder>DESIGNATION</zorder>
     <zorder>sendNotification</zorder>
     <zorder>textEdit</zorder>
     <zorder>pushsegment</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Mailing</string>
     </attribute>
     <widget class="QLineEdit" name="rcpt">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>190</y>
        <width>501</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLineEdit" name="subject">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>260</y>
        <width>501</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="msg">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>350</y>
        <width>511</width>
        <height>150</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sendBtn2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>560</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Envoyer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_54">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>270</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Sujet:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_56">
      <property name="geometry">
       <rect>
        <x>191</x>
        <y>386</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Message:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_53">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>190</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>-20</y>
        <width>1541</width>
        <height>1031</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/img/img/1.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_4</zorder>
     <zorder>rcpt</zorder>
     <zorder>subject</zorder>
     <zorder>msg</zorder>
     <zorder>sendBtn2</zorder>
     <zorder>label_54</zorder>
     <zorder>label_56</zorder>
     <zorder>label_53</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>-40</x>
        <y>0</y>
        <width>1781</width>
        <height>901</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
/* Style for QGroupBox */
QRadioButton{ color: white; /* Text color for labels */
    font-size: 16px; /* Adjust font size as needed */
    font-weight: bold; /* Make the text bold if desired */
    margin: 5px; /* Adjust margin as needed */
}

/* Style for QLineEdit (input box) */
QLineEdit {
    background-color: #3498db; /* Blue color for input boxes */
    border: 1px solid #2980b9; /* Border color for input boxes */
    border-radius: 5px; /* Border radius for input boxes */
    padding: 5px; /* Padding for input boxes */
    color: black; /* Text color inside input boxes */
}

/* Style for QLabel (text label) */
QLabel {
    color: white; /* Text color for labels */
    font-size: 16px; /* Adjust font size as needed */
    font-weight: bold; /* Make the text bold if desired */
    margin: 5px; /* Adjust margin as needed */
}

/* Style for QPushButton */
QPushButton {
    background-color: #3498db; /* Blue color for buttons */
    border: 1px solid #2980b9; /* Border color for buttons */
    border-radius: 5px; /* Border radius for buttons */
    padding: 5px 10px; /* Adjust padding as needed */
    color: white; /* Text color on buttons */
}

/* Additional styling for better appearance */
QPushButton:hover {
    background-color: #2980b9; /* Darker blue on hover */
}

QPushButton:pressed {
    background-color: #21618c; /* Even darker blue when pressed */
}

/* Add more styles as needed for your specific application */
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>290</y>
         <width>461</width>
         <height>161</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Camera</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>-150</y>
         <width>1391</width>
         <height>1031</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/img/img/1.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>label_13</zorder>
      <zorder>pushButton_3</zorder>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>audio</string>
     </attribute>
     <widget class="QGroupBox" name="verticalGroupBox">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>210</y>
        <width>550</width>
        <height>250</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="4">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="4">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="currentPosition">
         <property name="text">
          <string>00:00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="duration">
         <property name="text">
          <string>00:00</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QPushButton" name="playbutton">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="loop">
         <property name="text">
          <string>Loop</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QPushButton" name="selectbutton">
         <property name="text">
          <string>Select audio file</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="pauseButton">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QSlider" name="volume">
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QSlider" name="timeline">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="audioname">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-10</y>
        <width>1541</width>
        <height>1031</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/img/img/1.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_10</zorder>
     <zorder>verticalGroupBox</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1862</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
